src/app/page/UserProfile/UserOverview.jsx

```diff
@@ -15,12 +19,26 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const UserStatus = ({ view, profile, setProfile }) => {
+const UserStatus = ({ view, profile, setProfile, firestore }) => {
   const classes = useStyles();
+  // find the user
+  const user = useSelector((state) => state.firebase.auth);
 
   const displayName = _.get(profile, "displayName", "");
   const address = _.get(profile, "address", "");
 
+  useEffect(() => {
+    const userId = user.uid;
+    if (address) {
+      const location = addressLookUp(address);
+      location
+        .then((res) => firestore.collection("users").doc(userId).set({ location: res }))
```

Comment:
Does set() also update? I am guessing it does but wouldn't mind a verification.

Reply:
set does not update, it actually set the entire profile, so this would be, yeah, not good